### DISCUSSION TITLE
SVG icons in Custom Tkinter

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [TomSchimansky/CustomTkinter#2111] [Q&A]
How can I use SVG icons in Customtkinter? I saw that you first have to convert the svg to png, but how can I implement this in the code?

**Comment by ubaid2751** [comment]
Did you get any solution for this?

**Comment by Akascape** [comment]
@Heus-Sueh @ubaid2751

You can try this package: [Tksvg](https://github.com/TkinterEP/python-tksvg)

`svg_image = tksvg.SvgImage(file="test.svg")`

then use the svg image in the ctklabel.
